packages/kit/src/views/Swap/pages/components/LimitOrderCancelDialog.tsx

```
@@ -1,35 +1,28 @@
-      </SizableText>
+      <LimitOrderCard
+        item={item}
+        hiddenCancelIcon
+        hiddenHoverBg
+        progressWidth={gtMd ? 120 : 100}
+      />
+      <HyperlinkText
+        size="$bodyMd"
+        color="$textSubdued"
+        translationId={ETranslations.limit_cancel_order_off_chain_tip}
+      ></HyperlinkText>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)